ruleset.xml

```diff
@@ -5,5 +5,7 @@
 	<file>example.php</file>
 	<arg name="report" value="full"/>
 	<arg value="sp"/>
-	<rule ref="WordPress"/>
```

Comment:
Because I'm presuming that TGMPA will not be allowed to be used in VIP context and therefore (most of) the VIP rules don't apply.